dev/automated_tests/flutter_test/exception_handling_test.dart

```diff
@@ -10,7 +10,7 @@ void main() {
     throw 'Who lives, who dies, who tells your story?';
   });
   testWidgets('Exception handling in test harness - FlutterError', (WidgetTester tester) async {
-    throw FlutterError('Who lives, who dies, who tells your story?');
+    throw FlutterError.fromParts(<DiagnosticsNode>[ErrorSummary('Who lives, who dies, who tells your story?')]);
```

Comment:
would it make sense to put this and others back to just using the only-a-string constructor?
might make the PR easier to review.

Reply:
I am doing that right now. Splitting this CL into 3 more manageable CLs. All one line cases I've reverted.  First CL is breaking changes, 2nd CL is hints and cases with clear useful structure. 3rd CL is the borderline cases where it is totally fine to conclude that nothing should be done for many of the errors.